hack/generate.sh

```diff
@@ -121,13 +121,15 @@ jsonnet -c -J vendor -m monitoring-central/manifests \
         GCPExternalIpAddress: 'fake_external_ip_address',
         IAPClientID: 'fakeIAP_ID',
         IAPClientSecret: 'fakeIAP_secret',
+        issuer: 'issuer-123',
     },
     victoriametrics: {
         DNS: 'http://fake.victoriametrics.url',
         authKey: 'random-key',
         username: 'p@ssW0rd',
         password: 'user',
         GCPExternalIpAddress: 'fake_external_ip_address',
+        issuer: 'issuer-123',
```

Comment:
Hmmmmm, today we're using this to generate the YAML files that are imported in several places.

Did you run `make generate` to make sure it doesn't affect those files?

Reply:
Ah, just noticed this isn't touching any satellite code, just central :)

Good to go 🚀

Reply:
We use those generate ones for `monitoring-satellite`, not for `monitoring-central` - which this one generates